incubator/binding-mqtt-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/kafka/streams/kafka/session.will.message.close.deliver.will.message/client.rpt

```diff
@@ -0,0 +1,344 @@
+         option zilla:window 8192
+         option zilla:transmission "duplex"
+
+write zilla:begin.ext ${kafka:beginEx()
+                            .typeId(zilla:id("kafka"))
+                            .merged()
+                                .capabilities("FETCH_ONLY")
+                                .topic("mqtt_sessions")
+                                .filter()
+                                    .key("client-1#will-signal")
+                                    .build()
+                                .build()
+                            .build()}
+
+connected
+
+read advised zilla:flush
+
+write close
+read closed
+
+write notify LIFETIME_FETCHED
+
+connect await LIFETIME_FETCHED
+        "zilla://streams/kafka0"
+         option zilla:window 8192
+         option zilla:transmission "duplex"
+
+
+write zilla:begin.ext ${kafka:beginEx()
+                            .typeId(zilla:id("kafka"))
+                            .merged()
+                                .capabilities("PRODUCE_AND_FETCH")
+                                .topic("mqtt_sessions")
+                                .filter()
+                                    .key("client-1#migrate")
+                                    .headerNot("sender-id", "sender-1")
+                                    .build()
+                                .build()
+                            .build()}
+
+connected
+
+write zilla:data.ext ${kafka:dataEx()
+                           .typeId(zilla:id("kafka"))
+                           .merged()
+                               .deferred(0)
+                               .partition(-1, -1)
+                               .key("client-1#migrate")
+                               .hashKey("client-1")
+                               .header("sender-id", "sender-1")
+                               .build()
+                           .build()}
+write zilla:data.empty
+write flush
+write notify INITIAL_MIGRATE_SENT
+
+write close
+read closed
+
+
+connect await INITIAL_MIGRATE_SENT
+        "zilla://streams/kafka0"
+         option zilla:window 8192
+         option zilla:transmission "duplex"
+
+write zilla:begin.ext ${kafka:beginEx()
+                               .typeId(zilla:id("kafka"))
+                               .group()
+                                   .groupId("client-1")
+                                   .protocol("highlander")
+                                   .timeout(1000)
+                                   .build()
+                               .build()}
+
+connected
+
+read zilla:data.ext ${kafka:matchDataEx()
+                             .typeId(zilla:id("kafka"))
+                             .group()
+                                 .leaderId("member-1")
+                                 .memberId("member-1")
+                                 .members(1)
+                                 .build()
+                             .build()}
+read notify RECEIVED_LEADER_DATA
```

Comment:
Suggest rename to `RECEIVED_GROUP_MEMBERS_LEADER`.

In more complex scenarios, we can use `RECEIVED_GROUP_MEMBERS_NOT_LEADER` too.